dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1513,19 +1513,18 @@ class CompileTest {
         watch.start();
         await flutter('build', options: options);
         watch.stop();
-        final String basename = path.basename(cwd);
-        final String exePath = path.join(
+        final String buildPath = path.join(
           cwd,
           'build',
           'windows',
           'runner',
           'release',
-          '$basename.exe');
-        final File exe = file(exePath);
+        );
         // On Windows, we do not produce a single installation package file,
-        // rather a directory containing an .exe and .dll files.
-        // The release size is set to the size of the produced .exe file
-        releaseSizeInBytes = exe.lengthSync();
+        // rather a directory containing an .exe and .dll files. Zip them all
+        // together to get an approximate release size.
+        await exec('tar.exe', <String>['-zcf', 'build/app.tar.gz', buildPath]);
```

Comment:
Should we not track both?

Reply:
Oh, I see, this file is platform agnostic...

Reply:
Track both what?

The exe itself doesn't actually vary with dart code or engine size, its just the bootstrap. We need to measure exe + flutter_windows.dll + app.so + all assets